autoload/vimlparser.vim

```diff
@@ -3780,10 +3782,11 @@ function! s:StringReader.__init__(lines)
           if c == '\'
             let skip = s:FALSE
           endif
+          call add(self.buf, '<BOL>')
```

Comment:
[Question] What does "<BOL>" stand for? Beginning of Line?

Reply:
yes, added comment.

Reply:
Actually, it's not only the leading backslash, but also the spaces before the backslash.
Maybe we need to introduce different token type or rephrase the comment.

(I wander is BOL really good naming, it's true that the token is beginning of line but it doesn't imply it's only for line-continuation". But i couldn't come up with better naming and I couldn't find how original Vim source code mentions this leading backslash)